pkg/k8s/kubectl.go

```diff
@@ -0,0 +1,45 @@
+package k8s
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+
+	"k8s.io/apimachinery/pkg/version"
+)
+
+var minKubectlVersion = [3]int{1, 10, 0}
+
+type versionOutput struct {
+	ClientVersion version.Info
+}
+
+// CheckKubectlVersion validates whether the installed kubectl version is
+// running a minimum kubectl version.
+func CheckKubectlVersion() error {
+	cmd := exec.Command("kubectl", "version", "--client", "--output=json")
```

Comment:
Ok, though relying on `:` delimiter concerns me a bit. I've used regex to get version number (since `vX.Y.Z` seems to be a convention) and to make sure it works ok added test for it.``